items/FROZEN_STEVE_SC.json

```diff
@@ -69,6 +69,10 @@
           "id": "ICE_ROD",
           "chance": "4%"
         },
+        {
+          "id": "ESSENCE_ICE",
+          "chance": "x0-1"
```

Comment:
why isnt this a percent

Reply:
![{E417CEFB-034F-45B9-8907-A7D1245C75BB}](https://github.com/user-attachments/assets/213adf8d-1b4a-44f9-9464-8ba0b329e376)

wiki says 25% though so changed it, thanks